### DISCUSSION TITLE
Cards Image Resize?

### DISCUSSION THREAD

**Discussion by KDyulgerov** [GitbookIO/community#523] [2. Published content]
Before the update, the cards images was okay in phone version, now every image is resized individuality, will this be fixed or?
![image001](https://github.com/GitbookIO/community/assets/123773522/b601e383-b594-44bc-b52d-2ebf2ddc7362)

**Comment by emmerich** [comment]
Hi @KDyulgerov, we're aware of this issue and are working on fix to be deployed ASAP. Thanks again for letting us know.

**Reply by KDyulgerov** [reply]
Thank you for your quick response, have a great time✌️

**Reply by KDyulgerov** [reply]
When to expect the issue to be fiexed?

Already 3 weeks passed? @emmerich

**Comment by KDyulgerov** [comment]
I have another issue, when is on the draft, the hyperlink is okay with arrow. But when is live on the new version is like the second screenshot with no arrow and underlined text.
![image002](https://github.com/GitbookIO/community/assets/123773522/d88843e6-7a17-4048-b0e1-ee95d11ce62d)
